x-pack/plugins/security_solution/cypress/integration/data_sources/sourcerer.spec.ts

```diff
@@ -28,20 +31,33 @@ import { openTimelineUsingToggle } from '../../tasks/security_main';
 import { populateTimeline } from '../../tasks/timeline';
 import { SERVER_SIDE_EVENT_COUNT } from '../../screens/timeline';
 import { cleanKibana } from '../../tasks/common';
+import { createUsersAndRoles, secReadCasesAll, secReadCasesAllUser } from '../../tasks/privileges';
+import { TOASTER } from '../../screens/configure_cases';
 
+const usersToCreate = [secReadCasesAllUser];
+const rolesToCreate = [secReadCasesAll];
 // Skipped at the moment as this has flake due to click handler issues. This has been raised with team members
 // and the code is being re-worked and then these tests will be unskipped
-describe.skip('Sourcerer', () => {
-  before(() => {
+describe('Sourcerer', () => {
+  beforeEach(() => {
     cleanKibana();
   });
-
-  beforeEach(() => {
-    cy.clearLocalStorage();
-    loginAndWaitForPage(HOSTS_URL);
+  describe('permissions', () => {
+    before(() => {
+      createUsersAndRoles(usersToCreate, rolesToCreate);
+    });
+    it(`role(s) ${secReadCasesAllUser.roles.join()} shows error when user does not have permissions`, () => {
+      loginWithUserAndWaitForPageWithoutDateRange(HOSTS_URL, secReadCasesAllUser);
+      cy.get(TOASTER).should('have.text', 'Write role required to generate data');
+    });
   });
 
-  describe('Default scope', () => {
+  describe.skip('Default scope', () => {
```

Comment:
these were always skipped as they were flakey, added a test above for the permissions error from sourcerer so now skipping these individually